copier/main.py

```diff
@@ -513,6 +517,36 @@ def _render_file(self, src_abspath: Path) -> None:
             dst_abspath.write_bytes(new_content)
             dst_abspath.chmod(src_mode)
 
+    def _render_symlink(self, src_abspath: Path) -> None:
```

Comment:
Please add a docstring to the method, explaining clearly what it does. It will help my future self when maintaining this 😄